packages/aws-cdk-lib/aws-ecs/test/amis.test.ts

```diff
@@ -0,0 +1,23 @@
+import * as cdk from '../../core';
+import * as ecs from '../lib';
+
+describe('amis', () => {
+  test('BottleRocketImage with NVIDIA variant', () => {
+    // GIVEN
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app);
+
+    // WHEN
+    new ecs.BottleRocketImage({
+      variant: ecs.BottlerocketEcsVariant.AWS_ECS_1_NVIDIA,
+    }).getImage(stack);
+
+    /// THEN
+    const assembly = app.synth();
+    const parameters = assembly.getStackByName(stack.stackName).template.Parameters;
+    expect(Object.entries(parameters).some(
+      ([k, v]) => k.startsWith('SsmParameterValueawsservicebottlerocketawsecs1nvidiax8664') &&
+          (v as any).Default.includes('/bottlerocket/aws-ecs-1-nvidia/x86_64/'),
+    )).toEqual(true);
+  });
```

Comment:
```suggestion
  test.each([
    [ecs.BottlerocketEcsVariant.AWS_ECS_1, 'SsmParameterValueawsservicebottlerocketawsecs1nvidiax8664'],
    ...
  ])('BottleRocketImage with %s variant', (variant, ssmKey) => {
    // GIVEN
    const app = new cdk.App();
    const stack = new cdk.Stack(app);

    // WHEN
    new ecs.BottleRocketImage({
      variant,
    }).getImage(stack);

    /// THEN
    const assembly = app.synth();
    const parameters = assembly.getStackByName(stack.stackName).template.Parameters;
    expect(Object.entries(parameters).some(
      ([k, v]) => k.startsWith(ssmKey) && (v as any).Default.includes(`/bottlerocket/${variant}/x86_64/`),
    )).toEqual(true);
  });
```
Let's add a test for each variant.